appinventor/components/src/com/google/appinventor/components/runtime/util/WebRTCNativeMgr.java

```diff
@@ -441,18 +474,32 @@ public void run() {
             data.put("first", first);
             data.put("webrtc", true);
             data.put("key", rCode + "-r");
+            data.put("http11", true);
```

Comment:
What purpose does sending this value serve? We don't use it in the App Inventor side of things and for the rendezvous server you are also sending it in the query string above (but see my earlier comment).